rts/Sim/Projectiles/Projectile.cpp

```diff
@@ -122,7 +119,7 @@ void CProjectile::Init(const CUnit* owner, const float3& offset)
 	if (!weapon && !piece)
 		projectileHandler.AddProjectile(this);
 
-	if (synced && !weapon)
+	if (/*synced &&*/ !weapon)
```

Comment:
Is adding unsynced projectiles to the quadfield safe? Won't this disrupt iteration order or something.